src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,102 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page: number) => void,
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pagesCount = Array.from(
+    { length: Math.ceil(total / perPage) },
+    (_, i) => i + 1,
+  );
+
+  const isFirstPage = currentPage === pagesCount[0];
+  const isLastPage = currentPage === pagesCount[pagesCount.length - 1];
+
+  const handlePrevClick = (page: number) => {
+    if (!isFirstPage) {
+      onPageChange(page - 1);
+    }
+  };
+
+  const handleNextClick = (page: number) => {
+    if (!isLastPage) {
+      onPageChange(page + 1);
+    }
+  };
+
+  return (
+    <ul className="pagination">
+      <li
+        className={
+          cn(
+            'page-item',
+            { disabled: isFirstPage },
+          )
+        }
+        key="prevLink"
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={isFirstPage}
+          onClick={() => handlePrevClick(currentPage)}
```

Comment:
currentPage is a prop,  you can avoid putting it as a param here
```suggestion
          onClick={handlePrevClick}
```